staging_service/app.py

```diff
@@ -343,6 +343,10 @@ async def upload_files_chunked(request: web.Request):
         raise web.HTTPForbidden(  # forbidden isn't really the right code, should be 400
             text="cannot upload file with name beginning with space"
         )
+    if "," in filename:
+        raise web.HTTPForbidden(  # for consistency we use 403 again
+            text="cannot upload file with ',' in name"
+        )
```

Comment:
This is ok for now, but if we're going to make any more checks we should probably make a file checker function / regex / loop over a list of checkers that's defined at the module level or something